src/openrct2/ride/TrackPaint.h

```diff
@@ -84,7 +84,12 @@ constexpr uint8_t kTrackMap1x4[][4] = {
     { 0, 1, 2, 3 },
 };
 
-extern const int32_t DiagBlockedSegments[];
+const int32_t kDiagBlockedSegments[] = {
```

Comment:
This one was not made `constexpr`, was it intentional?

Reply:
Thanks for spotting that - not sure how I missed it...